src/main/resources/db/migration/V1_76__create_reasonable_adjustement_cjs_employer_indexes.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX juror_reasonable_adjustment_juror_number_idx ON juror_mod.juror_reasonable_adjustment (juror_number,reasonable_adjustment);
+CREATE INDEX juror_response_cjs_employment_cjs_employer_idx ON juror_mod.juror_response_cjs_employment USING btree (cjs_employer, juror_number);
```

Comment:
Can we also do the flip of this index so juror_number to cjs_employer (For when we are viewing a specific juror)

Reply:
yeah I'll add that now 😄